js/components/accessibility/autofocus.jsx

```diff
@@ -0,0 +1,55 @@
+/**
```

Comment:
why jsx?

Reply:
@ooyala-amohnatkin I changed it to "js", thank you.